wp-content/themes/oceanwp-child/functions.php

```diff
@@ -217,15 +296,27 @@ function custom_repeat_meta_box_save($post_id) {
 		$new = array();
 		$title = $_POST['UpdateTitle'];
         $date = $_POST['UpdateDate'];
-		$updates = $_POST['Update'];
-		 $count = count( $updates );
-		 for ( $i = 0; $i < $count; $i++ ) {
+		$updates = $_POST['Update'];		
+		$actiontype = $_POST['ActionType'];
+		$links = $_POST['ActionLink'];
```

Comment:
That's a bit strange to me. Can you explain how this POST data is coming? Aren't we using Ajax or getting data from wordpress DB?

Reply:
These are coming from custom meta boxes created for initiatives updates and action type. I had mentioned earlier that I have created updates function from the news meta boxes function you had created.